tests/test_bashcompletion.py

```diff
@@ -124,13 +124,13 @@ def my_fdopen(fd, mode, *args):
 
 # noinspection PyShadowingNames
 @pytest.mark.skipif(skip_no_argcomplete or skip_windows, reason=skip_reason)
-def test_invalid_ifs(parser1, mock):
+def test_invalid_ifs(parser1, mocker):
```

Comment:
Thanks for fixing the ``pytest`` warnings.  I also have no idea why they show up some of the time and not others!